.github/sync-repo-settings.yaml

```diff
@@ -113,6 +113,22 @@ branchProtectionRules:
       - cla/google
       - 'Kokoro - Test: Java GraalVM Native Image'
       - 'Kokoro - Test: Java 17 GraalVM Native Image'
+  - pattern: feat/transfer-manager
+    isAdminEnforced: true
+    requiredApprovingReviewCount: 1
+    requiresCodeOwnerReviews: true
+    requiresStrictStatusChecks: false
+    requiredStatusCheckContexts:
+    - dependencies (8)
```

Comment:
i don't think spacing before `-` has any effect on the parser reading but could you add padding similar to patterns above?

Reply:
good catch - didn't realize the spacing was misaligned!